templates/library/library-package.json

```
@@ -21,6 +21,7 @@
 
   "scripts": {
     "serve": "vue-cli-service serve dev/serve.<% if (ts) { %>ts<% } else { %>js<% } %>",
+    "prepublishOnly": "cross-env NODE_ENV=production rollup --config build/rollup.config.js",
```

Similar to the changes I'm proposing in https://github.com/team-innovation/vue-sfc-rollup/pull/87/files

why don't you reuse the same script instead? ;)

```suggestion
    "prepublishOnly": "npm run build",
```